src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1412,7 +1412,9 @@ private void getReadyToRunDelegateCtorHelper(ref CORINFO_RESOLVED_TOKEN pTargetM
             MethodDesc method = HandleToObject(pTargetMethod.hMethod);
             TypeDesc type = HandleToObject(delegateType);
 
-            DelegateCreationInfo delegateInfo = _compilation.GetDelegateCtor(type, method);
+            // TODO: we need a flag whether to do virtual resolution
+
+            DelegateCreationInfo delegateInfo = _compilation.GetDelegateCtor(type, method, followVirtualDispatch: true);
```

Comment:
@sandreenko If you add a flag on JitInterface to make it possible, you can use the flag to flip `followVirtualDispatch` to true or false.